case_studies/sdss_galaxies/config/config.yaml

```diff
@@ -99,6 +101,8 @@ models:
         psf_slen: 25
         sdss_bands:
             - 2
+        optimizer_params:
+            lr: 1e-3
```

Comment:
I think the `ImageDecoder` does not require `optimizer_params`?

Reply:
maybe you meant to include it within the `galaxy_model` block? (Although note that the `OneCenteredGalaxyAE` is not trained when within the decoder, it's kept fixed with the `ckpt` given)

Reply:
oops. fixed.